pulp_2_tests/tests/rpm/api_v2/test_republish.py

```diff
@@ -130,8 +130,9 @@ def test_cleanup_now(self):
         second publish. This test targets `Pulp #3816
         <https://pulp.plan.io/issues/3816>`_.
         """
-        if (self.cfg.pulp_version >= Version('2.17') and
-                not selectors.bug_is_fixed(3816, self.cfg.pulp_version)):
+        if self.cfg.pulp_version <= Version('2.17.1'):
```

Comment:
Since you're saying that the test requires `Pulp version 2.17.1 or newer`, then shouldn't the conditional be `if self.cfg.pulp_version < Version('2.17.1')`?